skeleton.html

```diff
@@ -32,6 +32,29 @@
       <button class="gameSpace -7"></button>
       <button class="gameSpace -8"></button>
     </div>
+    <div>
+    </br>
+      <form>
+        <input
+        type="text"
+        name="userName"
+        id="nameID"
+        placeholder="Enter Name"
+        />
+        <input
+        type="button"
+        id="submitButton"
+        value="Submit"
+        onclick="insertName()"
+        />
+      </form>
+      <span id="userName"></span>
+      <span id="somethingElse">RECORD: </span><span id="record"></span>
```

Comment:
you could just make this one span with an id of record
then when you change the innerHTML just make sure you add "RECORD: " + number

you could also do the same type of thing with the TURNS one below